plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesExt.kt

```diff
@@ -229,6 +231,19 @@ fun MapPluginExtensionsDelegate.setGesturesManager(
 fun MapPluginExtensionsDelegate.getGesturesSettings() =
   gesturesPlugin { getSettings() } as GesturesSettings?
 
+/**
+ * Update current gesture settings.
+ *
+ * Gesture plugin with id = [Plugin.MAPBOX_GESTURES_PLUGIN_ID] must be added while constructing
+ * [com.mapbox.maps.MapView] as part of [com.mapbox.maps.MapInitOptions.plugins].
+ *
+ * @throws IllegalStateException if gestures plugin was not added.
+ * @see [com.mapbox.maps.MapInitOptions]
+ */
+fun MapPluginExtensionsDelegate.updateGestureSettings(block: (GesturesSettings.() -> Unit)) {
```

Comment:
@kiryldz I'm ok with adding the extension here to expose the gesture setting updates to the MapboxMap, considering we already exposed a getter for the settings. 

But I would suggest the designed way to access gesture settings, which is `mapView.gestures.updateSettings()`, the reasons being: 
1. It's more consistent across all the plugins, where we don't expose the settings API on MapboxMap directly.
2. Since we generated the base configuration class that the plugin interface extends, any direct changes to the specific setting on the plugin interface can be applied automatically, comparing the getter extension here will only return a data class, and this could cause confusion to users like:
```
mapView.getMapboxMap().getGesturesSettings()?.focalPoint = ScreenCoordinate(
  mapView.width / 2.0,
  mapView.height / 2.0
)
```
will not work, since it won't be applied back to the gestures plugin, and 
```
mapView.gestures.focalPoint = ScreenCoordinate(mapView.width / 2.0, mapView.height / 2.0)
```
will work, since the generated config base class will apply the settings back to the plugin automatically.

Reply:
as discussed let's indeed deprecate current getter method to avoid confusion and add docs pointing to the correct way of updating gesture settings instead of introducing new update extension function